recipes/pyfab/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pyfab" %}
+{% set version = "0.3.0.post1" %}
+{% set sha256 = "4c6995f3f3eaadf01db680411fd417d0530de2848316b037a8b0965b79e27aac" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - nibabel
+    - numpy
+    - python
+    - six
+
+test:
+  imports:
+    - fabber
+
+about:
+  home: https://pyfab.readthedocs.io/
+  license: Granted by University of Oxford for use by academics carrying out research and not for use by consumers or commercial businesses. See file for more details
+  license_family: OTHER
+  license_file: 
```

Comment:
@chrisburr thanks for the feedback. We have been given tacit approval to distribute the software in this way but looking at the details of the license it looks like it may need modifying to reflect this. I will get back to the people who wrote/selected the license and see if we can get a clarification in there. 

The license file should definitely be in the PyPI tarball I will check this.

Reply:
@chrisburr I was just wondering if conda-forge has any specific policy on licensing terms acceptable for the repository?

I have contacted the relevant people and they are suggesting modifying the license to permit redistribution without condition and some form of usage, possibly for testing and evaluation purposes - i.e. almost a shareware-type license. Continued usage would then free for academic institutions but commercial organizations would need to obtain a commercial license from the university. 

Obviously this is not an open-source license in the standard sense but would it be acceptable for inclusion in conda-forge? Thanks for your help with this!

Reply:
@conda-forge/core The current licence for this package is [here](https://github.com/ibme-qubic/pyfab/blob/master/LICENSE). Can someone take a look and advise if changes are needed? If so, would the above proposal be okay?

Reply:
>  We have been given tacit approval to distribute the software in this way

@mcraig-ibme, who is we in this context? It is conda-forge / NumFOCUS needs permission.

Reply:
Basically, conda-forge / NumFOCUS needs at least an email from the copyright holder giving us permission to redistribute, and saying that this supersedes the license listed in with the code

Reply:
Thanks @chrisburr and @scopatz. I am the package author but the university is the copyright holder and determines the license. They have given permission for redistribution but I agree that this is not really consistent with the license as it is currently written. I am intending to ask for a change to the license to make redistribution rights clearer - I'd rather do this than give conda-forge an ad-hoc exception. Since it's a in-house written license it doesn't have a formal name (e.g. GPLv3) but we may be able to give it one as the same license will apply to a number of other products.

I will update the PR when this has been done. Thanks again for all your help.

Reply:
OK sounds good! We'll wait on this then